src/frontend/src/flows/authorize/index.ts

```diff
@@ -128,3 +148,49 @@ export const authFlowAuthorize = async (
       break;
   }
 };
+
```

Comment:
All of this chasm stuff was simply moved here from `authenticateBox` since the other authn flow (authn to manage) doesn't care about chasm (alternative origin)